app.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios");
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
Check JS conventions.

I thought there is supposed to be a blank line between requirements and actual code.

Reply:
After browsing popular repos, you are right about that. Done.